tools/jlink-nuttx.c

```diff
@@ -72,20 +91,24 @@ enum symbol_e
   NSYMBOLS
 };
 
-__attribute__ ((packed)) struct tcbinfo_s
+begin_packed_struct
+struct tcbinfo_s
 {
   uint16_t pid_off;
   uint16_t state_off;
   uint16_t pri_off;
   uint16_t name_off;
   uint16_t reg_num;
+  begin_packed_struct
```

Comment:
let's add the same change to device side too

Reply:
Yes. I think this should be done with this PR

Reply:
Yes, I agree.